diff --git a/run.sh b/run.sh
index c8ce535..9ec6cf4 100644
--- a/run.sh
+++ b/run.sh
@@ -1,2 +1,2 @@
-python train.py --debug True --save_dir "$1" --epochs $2 --run_name seresnext50 --model_name se_resnext50_32x4d --rgb True --activation mish --mixup True --cutmix True --continue_train True
+python train.py --verbose False --save_dir "$1" --epochs $2 --run_name seresnext50 --model_name se_resnext50_32x4d --rgb True --activation mish --mixup True --cutmix True --continue_train True
 # python train.py --epochs 20 --run_name seresnext50 --model_name efficientnet-b0 --rgb True --activation mish --debug True
